packages/plugins/plugin-assistant/src/components/ChatThread/ChatThread.stories.tsx

```diff
@@ -232,11 +238,20 @@ const meta = {
   component: ChatThread,
   render: render(StoryContainer),
   decorators: [
-    withClientProvider({
-      createIdentity: true,
-      types: [DataType.Organization, DataType.Person],
+    withPluginManager({
```

Comment:
is it possible to just use `withClientProvider`; the plugin manager is really heavy. or do we need the full intent stack? @wittjosiah?